src/lua/luaTask.c

```diff
@@ -37,14 +37,20 @@
 #include "lua.h"
 #include "lauxlib.h"
 #include "lualib.h"
+#include "GPIO.h"
 
 #include <stdbool.h>
 
+#define ERROR_LED 3
 #define DEFAULT_ONTICK_HZ 1
 #define MAX_ONTICK_HZ 30
 #define LUA_STACK_SIZE 1000
 #define LUA_PERIODIC_FUNCTION "onTick"
 
+#define LUA_BYPASS_FLASH_DELAY 250
+#define LUA_BYPASS_DELAY_SEC 5
+#define LUA_BYPASS_FLASH_COUNT (1000 / LUA_BYPASS_FLASH_DELAY) * LUA_BYPASS_DELAY_SEC
```

Comment:
Well what about how many times you go through that loop? Won't it be different on MK1 vs MK2?